protostar/starknet_gateway/account_manager.py

```diff
@@ -0,0 +1,62 @@
+from starknet_py.net.signer import BaseSigner
+from starknet_py.net import AccountClient, KeyPair
+from starknet_py.net.gateway_client import GatewayClient
+from starknet_py.net.client_models import Call as SNCall
+
+from protostar.starknet import Address
+from protostar.starknet_gateway.gateway_facade import Fee
+from protostar.starknet_gateway.multicall.multicall_protocols import (
+    UnsignedMulticallTransaction,
+)
+
+from .multicall import SignedMulticallTransaction, MulticallAccountManagerProtocol
+
+
+class AccountManager(MulticallAccountManagerProtocol):
+    def __init__(
+        self,
+        private_key: int,
+        address: Address,
+        signer: BaseSigner,
```

Comment:
I saw that those three are also used in DevnetAccount - maybe it's worth to introduce this account structure somehow to our production code as well?

Reply:
I think `AccountManager` is sufficient. `DevnetAccount` is a data structure which works well for tests, but production code needs an object.

Reply:
NVM. I added this data structure to reduce boilerplate in tests. `DevnetAccount` will inherit from `Account` in a different PR because there's a mismatch between types of private keys and I don't want to do the integration in this PR.

[use account data structure](https://github.com/software-mansion/protostar/pull/1212/commits/1ea413230fc29fb7b8afb258c8d36be3c9bcb34e)